examples/decimal-type/decimal_type.md

```diff
@@ -1,11 +1,6 @@
 # Decimal type
 
-The `decimal` type is the third numeric type. It works like `int` and `float` types.
-It is a separate basic type and belongs to the `anydata` type.
-There is no implicit conversions between `decimal` and other numeric types.
-It can represent `decimal` fractions exactly and it preserves precision.
-The `decimal` type does not include infinity, NaN, or negative zero.
-It supports floating-point precision to 34 decimal digits.
+The `decimal` type is the third numeric type. It works like `int` and `float` types. It is a separate basic type and belongs to the `anydata` type. There is no implicit conversions between `decimal` and other numeric types. It can represent `decimal` fractions exactly and it preserves precision. The `decimal` type does not include infinity, NaN, or negative zero. It supports floating-point precision to 34 decimal digits.
```

Comment:
```suggestion
The `decimal` type is the third numeric type. It works like `int` and `float` types. It is a separate basic type and belongs to the `anydata` type. There are no implicit conversions between `decimal` and other numeric types. It can represent `decimal` fractions exactly and it preserves precision. The `decimal` type does not include infinity, NaN, or negative zero. It supports floating-point precision to 34 decimal digits.
```